src/easydmp/eestore/migrations/0002_convert_to_jsonb.py

```diff
@@ -0,0 +1,39 @@
+# Generated by Django 2.2.12 on 2020-10-16 10:27
+
+import django.contrib.postgres.fields.jsonb
+import django.core.serializers.json
+from django.db import migrations
+
+import json
+
+
+def convert_to_jsonblob(apps, schema_editor):
+    EEStoreCache = apps.get_model('eestore', 'EEStoreCache')
+    for ec in EEStoreCache.objects.all():
+        if isinstance(ec.data, str):
+            ec.data = json.loads(ec.data)
+            ec.save()
+
+
+def convert_to_text(apps, schema_editor):
+    "Will be converted back by previous migration in operations list"
```

Comment:
Hm is that legal. :)

Reply:
If you migrate back to the migration before this one (0001_something) the field will be text in the database again. Tested :9 Only works when the two field types are comnpatible. Good thing too, as the "correct" way is very slow on the eventlog!